### ISSUE TITLE
Missing vacuum status and fan speed in display

### ISSUE THREAD

**Issue by idogrf** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#610]
### Checklist

- [X] I have updated the card to the latest version available
- [X] I have cleared the cache of my browser
- [X] I have checked if the problem is already reported

### The problem

Ever since updating to v2.2.0 I'm missing the Status and Fan Speed in the display
My vacuum is the 1gen xiaomi mi robot

![image](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/15867904/c67adb30-1bd8-409d-a5df-ed1841c1e9d4)

### What version of a card has described problem?

v2.2.1

### What was the last working version card?

v2.1.2

### What vacuum model do you have problems with?

xiaomi mi robot (1st gen)

### Which integration do you use to control your vacuum (link)?

https://www.home-assistant.io/integrations/xiaomi_miio/

### What browser (browsers/apps) does have this problem?

Chrome/Brave

### What version of Home Assistant do you use?

2022.8.3

### What type of installation are you running?

Home Assistant Container

### Card's configuration

```yaml
type: custom:xiaomi-vacuum-map-card
map_source:
  camera: camera.xiaomi_cloud_map_extractor
calibration_source:
  camera: true
entity: vacuum.xiaomi_vacuum_cleaner
vacuum_platform: default
```

**Comment by PiotrMachowski** [comment]
Have you tried clearing cache? Does anything happen when you click the tile?

**Comment by idogrf** [comment]
Yes, I tried clearing the cache
I also tried different devices and I get the same results on 2 pcs and an android

Nothing happens when I click the tile - the animation plays but nothing opens

**Comment by PiotrMachowski** [comment]
I have noticed that you use quite old HA version, I suspect that this might be a reason of your problems

**Comment by PiotrMachowski** [comment]
Fixed in v2.2.2

**Comment by idogrf** [comment]
Amazing, v2.2.2 did the trick!
Thanks @PiotrMachowski for the quick response
